test/integration/api-create.test.js

```diff
@@ -993,3 +992,363 @@ test('run external network (docker run --label triton.network.public=)',
+            // Note: Error is already checked in createDockerContainer
+            assert.object(err, 'err');
+            t.end();
+        }
+    });
+
+    // privision when there is correct owner
+    tt.test('run with assigned ipv4 address with correct owner', function (t) {
+        var assignedAddr = '10.0.21.200';
+        h.createDockerContainer({
+            vmapiClient: VMAPI,
+            dockerClient: DOCKER_ALICE,
+            test: t,
+            extra: {
+                'HostConfig.PublishAllPorts': true,
+                Labels: {
+                    'triton.network.public': externalNetworkOwner.name,
+                    'triton.network.public_ipv4': assignedAddr
+                }
+            },
+            start: true
+        }, oncreate);
+
+        function oncreate(err, result) {
+            assert.strictEqual(err, null);
+            var nics = result.vm.nics;
+            var extNic = nics[0];
+            if (FABRICS) {
+                extNic = (nics[0].primary === true ? nics[0] : nics[1]);
+                t.equal(nics.length, 2, 'one nics');
+            } else {
+                t.equal(nics.length, 1, 'one nic');
+            }
+            t.equal(extNic.ip, assignedAddr, 'correct external ip')
+            DOCKER_ALICE.del('/containers/' + result.id + '?force=1', ondelete);
+        }
+
+        function ondelete(err) {
+            t.ifErr(err, 'delete external triton.network.public_ipv4 testing container');
+            t.end();
+        }
+    });
+
+    // fail to privision when there is incorrect owner
+    tt.test('run with assigned ipv4 address with incorrect owner', function (t) {
+        var assignedAddr = '10.0.41.200';
+        var expectedErr = '(Error) network sdcdockertest_apicreate_external_bob0 '
+          + 'not found';
+        h.createDockerContainer({
+            vmapiClient: VMAPI,
+            dockerClient: DOCKER_ALICE,
+            test: t,
+            expectedErr: expectedErr,
+            extra: {
+                'HostConfig.PublishAllPorts': true,
+                Labels: {
+                    'triton.network.public': externalNetworkWrongOwner.name,
+                    'triton.network.public_ipv4': assignedAddr
+                }
+            },
+            start: true
+        }, oncreate);
+
+        function oncreate(err, result) {
+            // Note: Error is already checked in createDockerContainer
+            assert.object(err, 'err');
+            t.end();
+        }
+    });
+
+    // privision when there is multiple ip on same network
+    // with correct owner
+    tt.test('run with assigned ipv4 address with multiple ip ' +
+        'on same network with correct owner', function (t) {
+        if (!FABRICS) {
+            t.skip('sdc-docker without fabrics currently do ' +
+                'not support multiple networks')
+            return t.end()
+        }
+        var assignedAddrLabel = '10.0.21.200';
+        var assignedAddrClient = '10.0.21.201';
```

Comment:
Per what we discussed, this is a good idea to skip per above.  Mentioning this here because I'm gonna ask something else down further.

Reply:
The passing tests suggest we're doing this correctly now, right?